src/main/java/org/mifos/processor/bulk/camel/config/HttpClientConfigurerTrustAllCACerts.java

```diff
@@ -0,0 +1,73 @@
+package org.mifos.processor.bulk.camel.config;
+
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLContext;
+import org.apache.camel.component.http.HttpClientConfigurer;
+import org.apache.http.config.Registry;
+import org.apache.http.config.RegistryBuilder;
+import org.apache.http.conn.socket.ConnectionSocketFactory;
+import org.apache.http.conn.socket.PlainConnectionSocketFactory;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.ssl.SSLContextBuilder;
+import org.apache.http.conn.ssl.TrustStrategy;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HttpClientConfigurerTrustAllCACerts implements HttpClientConfigurer {
```

Comment:
can we create this as a spring bean,  we can make sure that this will be a singleton.

Reply:
This is a configuration client used only in a single place. Why we need to create a bean?

Reply:
Will multiple thread access the same code and create multiple instances of the same configuration. Ideally that is not needed. So it's safe to create it as singleton bean

Reply:
ok will do that.

Reply:
done